.github/workflows/release.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         working-directory: build
         run: |
           cmake --version
-          cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DCPACK_GENERATOR="DEB" -DPRECICE_PETScMapping=ON -DBUILD_TESTING=OFF -DBUILD_SHARED_LIBS=ON ..
+          cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DCPACK_GENERATOR="DEB" -DPRECICE_FEATURE_PETSC_MAPPING=ON -DBUILD_TESTING=OFF -DBUILD_SHARED_LIBS=ON ..
```

Comment:
I'm not sure about benefits, but we could consider enabling `CMAKE_INTERPROCEDRUAL_OPTIMIZATION` for the package generation (we also highlight this in our CMake variables).

Reply:
True.

When we drop Ubuntu 20.04, our new cmake baseline is 3.22 and then we can use a preset for this, which were introduced with version 3.19.

Reply:
Still a valuable switch. Maybe also for the request debian packages workflow..